zhaquirks/tuya/ts004f.py

```diff
@@ -135,31 +135,31 @@ class TuyaSmartRemote004FROK(CustomDevice):
             COMMAND: COMMAND_MOVE_SATURATION,
             ENDPOINT_ID: 1,
             CLUSTER_ID: 768,
-            ARGS: [1, 200],
+            PARAMS: {"saturation": 1, "transition_time": 200},
```

Comment:
per the testing from @MattWestb this should be `move_mode`

Reply:
I think `rate": 200` is the failing part.

Reply:
![Screen Shot 2022-04-09 at 10 00 08 AM](https://user-images.githubusercontent.com/1335687/162577507-b8971dbe-378a-4974-b2a9-045fd0a26211.png)

looks like move to saturation VS move saturation

Reply:
Looks like quirks doesn't have a `move_saturation` constant at all, only `move_to_saturation`...

Reply:
Or not, that's the only place any `_saturation` command is used.